Realm/RLMObjectStore.mm

```diff
@@ -435,10 +432,13 @@ void RLMAddObjectToRealm(__unsafe_unretained RLMObjectBase *const object,
 
     // get or create row
     bool created;
-    auto primaryGetter = [=](__unsafe_unretained RLMProperty *const p) {
-        return [object valueForKey:p.getterName];
-    };
-    object->_row = (*schema.table)[RLMCreateOrGetRowForObject(schema, primaryGetter, options, created)];
+    auto primaryGetter = [=](RLMProperty *p) { return [object valueForKey:p.getterName]; };
```

Comment:
I made this change in the other spot RLMCreateOrGetRowForObject was called...

Will make a new pr with this fix.